client/consensus/slots/src/slots.rs

```diff
@@ -124,24 +124,20 @@ where
 	/// Returns a future that fires when the next slot starts.
 	pub async fn next_slot(&mut self) -> Result<SlotInfo<Block>, Error> {
```

Comment:
This doesn't need to return `Result` anymore. We don't return any error somewhere.